### PR TITLE
Added new release script

### PR DESCRIPTION
**Pull request by Daniel-Sanche**
Final issue to complete improved release plan (https://github.com/GoogleCloudPlatform/stackdriver-sandbox/issues/114)

I added a release script, and updated RELEASING.md to add instructions on how to use it

For more detailed information, read and comment on RELEASING.md

### CODE REVIEW

File: make-release.sh [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+fi
+
+# temporarily pin manifests to :$NEW_VERSION
+find "${REPO_ROOT}/kubernetes-manifests" -name '*.yaml' -exec sed -i -e "s/:latest/:${NEW_VERSION}/g" {} \;
+
+# update website deployment tag
+sed -i -e "s/cloudshell_git_branch=v\([0-9\.]\+\)/cloudshell_git_branch=v0.2.0/g" ${REPO_ROOT}/docs/index.html;
+
+if [[ "$*" == *dryrun*  || "$*" == *dry-run* ]]; then
+    exit 0
```
Comment (simonz130) [inline make-release.sh:37]: Maybe add some printed statements to clarify that no release will happen due to --dry-run option

File: make-release.sh [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    echo "${NEW_VERSION} argument must conform to regex string:  ^v[0-9]+\.[0-9]+\.[0-9]+$ "
+    echo "ex. v1.0.1"
+    exit 1
+fi
+
+# temporarily pin manifests to :$NEW_VERSION
+find "${REPO_ROOT}/kubernetes-manifests" -name '*.yaml' -exec sed -i -e "s/:latest/:${NEW_VERSION}/g" {} \;
+
+# update website deployment tag
+sed -i -e "s/cloudshell_git_branch=v\([0-9\.]\+\)/cloudshell_git_branch=v0.2.0/g" ${REPO_ROOT}/docs/index.html;
```
Comment (simonz130) [inline make-release.sh:34]: instead of v0.2.0 it should be ${NEW_VERSION}

Comment (Daniel-Sanche) [reply make-release.sh:34]: good catch!

File: make-release.sh (hunk 1 above)
Comment (Daniel-Sanche) [reply make-release.sh:37]: done

**Pull request merged**